spec.md

```diff
@@ -164,49 +205,105 @@ echo "SPDX-License-Identifier: MIT" > file.txt
 # REUSE-IgnoreEnd
 ```
 
-### In-line snippet comments
+### REUSE.toml
 
-If a copyright and/or licensing info is to apply only to a certain snippet
-instead of the whole file, SPDX snippet tags SHOULD be used (as defined in [SPDX
-Specification, Annex H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
+Copyright and Licensing Information MAY be associated with a file through a
+`REUSE.toml` file. The intended use case of this method is large directories where
+including a comment header in each file (or in `.license` companion files) is
+impossible or undesirable.
 
-Such an annotated snippet block MUST start with `SPDX-SnippetBegin` to mark its
-beginning and end with `SPDX-SnippetEnd` to mark the snippet's end.
+A `REUSE.toml` file MAY be located in any directory, and can cover files that are
+within its directory or deeper. You MAY have multiple `REUSE.toml` files in
+different directories.
 
-Do note that SPDX snippet tags MUST start with `SPDX-Snippet`, meaning that the
-correct copyright notice in a snippet is `SPDX-SnippetCopyrightText`.
+The `version` key (REQUIRED) MUST have an integer value representing the schema
+version of the file. This specification describes version 1 of `REUSE.toml`.
```

Comment:
Do we anticipate other version numbers, like `1.1` or even alpha releases to try new things?

Reply:
I think doing semantic versioning here is too granular. I don't expect this version number to bump often or at all. Even if we add new (optional) keys, we don't need to bump this version, I think.

For alpha releases (which we don't really do anyway), we can just pick a number and not really care.

Also, because this is TOML, you can just add your own keys for semantics undefined by the spec. I will add a small paragraph about that.

Reply:
Agreed, just wanted to be sure.